lancache-litmus/overlay/var/www/litmus/index.html

```diff
@@ -1,208 +1,697 @@
 </head>
 <body>
-	<div class="container-fluid">
-		<div class="jumbotron margin-fix">
-			<div class="container">
-				<h1 class="display-4">Lancache Litmus Test</h1>
-				<p class="lead">We are aware of <b id="domaincount">0</b> domains spanning <b id="providercount">0</b> providers.</p>
-				<p id="urltext"></p>
-			</div>
-		</div>
-		<br>
-		<div id="cdns" class="row">
-			<div id="cdn-tester" class="cdnentry col-md-12">
-				<div class="item text-center border rounded alert alert-secondary">
-					<div class="form-group margin-fix">
-						<div class="col-sm-6 offset-sm-3">
-							<div class="input-group">
-								<input type="text" class="form-control rounded" id="inputTest" placeholder="Check domain (Wildcards valid)">
-								<span class="input-group-btn">
-									&nbsp;
-									<button id="inputTestButton" class="btn btn-primary mb-2">Test</button>
-							   </span>
-							</div>
-						</div>
-					</div>
-				</div>
-			</div>
-			<div class="cdnentry col-md-12">
-				<div id="cache-info" class="item text-center border rounded alert alert-warning">
-					<button type="button" class="close" aria-label="Close">
-						<span aria-hidden="true" onclick="hideAlert()">&times;</span>
-					</button>
-					<span id="cache-info-display-title">
-						<b>The following domains did not resolve to a LanCache:</b><br>
-					</span>
-					<span id="cache-info-display">
-					</span>
-				</div>
-			</div>
-			<div id="cdntemplate" class="cdnentry col-md-3">
-				<div class='item text-center border rounded alert-info'>
-					<b><span id="title">Nothing is defined?<br>I'm not sure how you've done this, but you have.</span></b><br>
-					<span id="tagline">So well done.</span><br>
-					<span id="workingcdndomaincount">0</span>/<span id="totalcdndomaincount">0</span> domains working
-				</div>
-			</div>
-		</div>
-	</div>
+    <div class="container">
+        <!-- Header Section -->
+        <div class="header">
+            <img src="https://lancache.net/img/lancache-fulllogo.svg" alt="Logo">
+            <h1>LanCache Test</h1>
+            <p class="subtitle">
+                We are aware of <strong id="domaincount">0</strong> domains spanning 
+                <strong id="providercount">0</strong> providers.
+            </p>
+            
+            <div class="warning-banner">
+                🦊 Test this website in the Mozilla Firefox browser if the site doesnt work at all
+            </div>
+            
+            <div class="dns-info">
+                <p style="margin-bottom: 1rem;">During LanCache Test if you find the listed Gaming Stream sites RED then your Cache Services are NOT working. Get started with LanCache today!</p>
+                
+                <div class="button-grid">
+                    <a href="https://lancache.net/" class="info-button" target="_blank" rel="noopener">
+                        🚀 Learn more about Lancache
+                    </a>
+                </div>
+            </div>
+        </div>
```

Comment:
```suggestion
            </div>
            <p id="urltext"></p>
        </div>
```

Reinsert `urltext` to for page.